test/info.py

```diff
@@ -85,6 +86,7 @@
     (   0x1114,     'lpc11u35_ssci1114_if',                     None,               'LPC1114FN28'                           ),
     (   0x1120,     'sam3u2c_mkit_dk_dongle_nrf5x_if',          'sam3u2c_bl',       'Nordic-nRF51-Dongle'                   ),
     (   0x1234,     'lpc11u35_c027_if',                         None,               'u-blox-C027'                           ),
+    (   0x1238,     'kl26z_nina_b1_if',                        'kl26z_bl',          'u-blox-NINA-B1'                        ),
```

Comment:
Ok done, the PR for UBLOX_EVA_NINA board is already pending in mbed-os [link](https://github.com/ARMmbed/mbed-os/pull/3291)